polaris-react/src/components/Navigation/Navigation.scss

```
@@ -281,7 +268,15 @@ $disabled-fade: 0.6;
   word-break: break-all;
 }
 
+.SecondaryActions {
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
```

```suggestion
  // stylelint-disable-next-line
```

Remove the comment that it was added by the migrator since this wasn't.

---

Yup I can remove them. I wasn't sure if we would "automate" out those line later and if would clean it up without the notes :).

---

At this point we are just trying to unblock adding the linter despite all the warnings. Eventually we would expect they will be cleaned up though I don't know that it will be done automagically because they are quite context specific.